plugins/carbon_black_defense/plugin.spec.yaml

```diff
@@ -523,184 +135,388 @@ types:
+      description: "Device name"
+      required: false
+    backend_timestamp:
+      title: "Backend Timestamp"
+      type: string
+      description: "Backend timestamp"
+      required: false
+    parent_pid:
+      title: "Parent PID"
+      type: integer
+      description: "Parent PID"
+      required: false
+    device_policy:
+      title: "Device Policy"
+      type: string
+      description: "Device policy"
+      required: false
+    parent_reputation:
+      title: "Parent Reputation"
+      type: string
+      description: "Parent reputation"
+      required: false
 connection:
   url:
     title: URL
     type: string
     description: API URL
-    default: https://api.confer.net
+    default: https://defense.conferdeploy.net
     required: true
+    example: https://example.com
   api_key:
     title: API Key
     type: credential_secret_key
     description: API key
     required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  org_key:
+    title: Organization Key
+    type: string
+    description: Organization key
+    required: true
+    example: 1ABCDEFGH
   connector:
     title: Connector
     type: string
     description: Connector ID
     required: true
+    example: 1ABCD23333
 actions:
   find_event:
     title: Find Event
     description: Retrieves all events matching the input search criteria. Response
       is a list of events in JSON format. Resulting events are sorted in descending
       order of time
     input:
-      hostName:
-        title: Hostname
-        description: Filter on hostnames case insensitive. For example hostName=win-IA9NQ1GN8OI
-          will match the hostname WIN-IA9NQ1GN8OI
-        type: string
-        required: false
-      hostNameExact:
-        title: Hostname Exact
-        description: Filter on the exact hostname. For example hostName=WIN-IA9NQ1GN8OI
-          will only return devices with the exact hostname WIN-IA9NQ1GN8OI but not
-          a host named win-IA9NQ1GN8OI
-        type: string
-        required: false
-      ownerName:
-        title: Owners Name
-        description: Filter on owner name case insensitive
+      device_name:
+        title: "Device Name"
+        description: "Device name"
         type: string
         required: false
-      ownerNameExact:
-        title: Owners Name Exact
-        description: Same as 'Owners Name' but with case sensitivity
+        example: "CONTOSO\\bruce-win10"
+      device_installed_by:
+        title: "Device Installed By"
+        description: "Device installed by"
         type: string
         required: false
-      ipAddress:
-        title: IP Address
-        description: Filter on events generated by a device with a given external
-          or internal IP address
+        example: "CONTOSO\\admin"
+      device_external_ip:
+        title: "Device External IP"
         type: string
+        description: "Device external IP"
         required: false
-      sha256hash:
+        example: "45.62.187.4"
```

Comment:
Let's change this to a generic IP from the style guide https://docs.rapid7.com/insightconnect/style-guide

Reply:
Changed to generic IP